lib/console/blockchain_api.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Console.BlockchainApi do
+  use HTTPoison.Base
+  require HTTPoison.Retry
+  alias HTTPoison.Response
+  alias HTTPoison.Retry
+
+  def each_page(url, fun, cursor \\ nil) do
```

Comment:
so we update the db 1 page at a time right? but this is still running on a single thread? do you think it's worthwhile to start a new task per page?